samples/tfm_integration/psa_crypto/README.rst

```diff
@@ -197,6 +197,13 @@ We need to reset the board manually after flashing the image to run this code.
 On nRF5340 and nRF9160:
 =======================
 
+.. note::
+   To be able to create a initial attestation token (IAT) the device has to be provisioned,
+   which is done by the :ref:`provisioning_image` sample.
```

Comment:
See if writing the reference like this will fix the build:
:ref:`nrf:provisioning_image`

Reply:
Sadly that didn't work